Modules/Filtering/ImageGrid/include/itkResampleImageFilter.h

```diff
@@ -220,7 +220,7 @@ class ITK_TEMPLATE_EXPORT ResampleImageFilter : public ImageToImageFilter<TInput
   itkGetConstReferenceMacro(Size, SizeType);
   /** @ITKEndGrouping */
   /** Get/Set the pixel value when a transformed pixel is outside of the
-   * image.  The default default pixel value is 0. */
+   * image.  The default pixel value is 0. */
```

Comment:
Maybe this was intended: the default of `DefaultPixelValue` is zero. So maybe that's the _default default_ pixel value 🤷